src/autopas/molecularDynamics/LJFunctor.h

```diff
@@ -527,27 +616,27 @@ class LJFunctor
         size, soa.template begin<Particle::AttributeNames::forceZ>());
 
 #else
-    utils::ExceptionHandler::exception("AutoPas was compiled without CUDA support!");
+    utils::ExceptionHandler::exception("LJFunctor::deviceSoAExtractor: AutoPas was compiled without CUDA support!");
 #endif
   }
 
   /**
    * @copydoc Functor::getNeededAttr()
    */
-  constexpr static const std::array<typename Particle::AttributeNames, 8> getNeededAttr() {
-    return std::array<typename Particle::AttributeNames, 8>{
+  constexpr static const std::array<typename Particle::AttributeNames, 9> getNeededAttr() {
```

Comment:
```suggestion
  constexpr static const auto getNeededAttr() {
```

Reply:
either this or remove the `std::array<typename Particle::AttributeNames, 9>` in the return statement